ims/common/config.py

```diff
@@ -73,5 +114,6 @@ def section(self, section_name, required=True):
 
 
 class ConfigSection:
+    """ A Simple Object to be used to get cfg.section.option """
     def __init__(self):
```

Comment:
I thought you wanted a dummy object for making changes. If you have intention of using this later, please keep it with some documentation. (but again there are other ways to do this).